### ISSUE TITLE
TypeError: type.getFields is not a function

### ISSUE THREAD

**Issue by mlegenhausen** [avantcredit/gql2ts#71]
I used 1.0.2. The problem only occurs when I install `gql2ts` with `npm i gql2ts` without `-g` option.

```
node_modules/@gql2ts/from-schema/dist/index.js:120
        const f1 = type.getFields();
                        ^

TypeError: type.getFields is not a function
    at typeToInterface (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:120:25)
    at typeArr.filter.filter.map.type (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:156:26)
    at Array.map (native)
    at typesToInterfaces (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:156:14)
    at run (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:162:12)
    at Object.exports.schemaToInterfaces (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:164:63)
    at Object.exports.generateNamespace (/xxx/node_modules/@gql2ts/from-schema/dist/index.js:167:85)
    at run (/xxx/node_modules/gql2ts/dist/index.js:38:35)
    at Socket.process.stdin.on (/xxx/node_modules/gql2ts/dist/index.js:54:35)
    at emitNone (events.js:110:20)
```

**Comment by brettjurgens** [comment]
so it works when installing globally?

that's strange. would it be possible to get a minimal reproducible example?

**Comment by epicallan** [comment]
I am having this same exact issue.

**Comment by epicallan** [comment]
It seems this issue was due to missing peer dependencies for me or incompatible dependencies.

**Comment by brettjurgens** [comment]
@mlegenhausen if you remove `npm_modules` and reinstall, does it work? do you have `@gql2ts/from-schema` in your package.json with a lower version?

**Comment by zamiang** [comment]
Getting the same issue using yarn and struggling to resolve. 

If I rm `yarn.lock` and all `node_modules` then re-install, I get this error every time. Tried pinning to 1.0.0, 1.0.1, 1.0.2 and 1.0.3 (which doesn't have corresponding commits?). Agree with @epicallan that it is something with incompatible dependencies.

**Comment by zamiang** [comment]
Fix for me was to upgrade `gql2ts` to 1.0.3 and `graphql` to 0.10.x (from 0.9.x in my case)

**Comment by mlegenhausen** [comment]
@brettjurgens problem is fixed in in `1.0.3`. Thanks 😄
